### ISSUE TITLE
On dual screen move hot corners to the extended screen edges

### ISSUE THREAD

**Issue by memeplex** [linuxmint/Cinnamon#7726]
```
~:: cinnamon --version
Cinnamon 3.8.6
~:: uname -a
Linux carlos 4.14.53-1-MANJARO #1 SMP PREEMPT Tue Jul 3 17:59:17 UTC 2018 x86_64 GNU/Linux
~:: lspci | grep VGA
00:02.0 VGA compatible controller: Intel Corporation 2nd Generation Core Processor Family Integrated Graphics Controller (rev 09)
~:: pacman -Qs video intel
local/xf86-video-intel 1:2.99.917+829+gd7dfab62-1 (xorg-drivers)
    X.org Intel i810/i830/i915/945G/G965+ video drivers
 ```

I have an external monitor on top of my laptop screen. The hot corners in my desktop are in the left and right top. But it's very hard to put the mouse pointer over them since the pointer passes through to the top screen. Now, since both screens form an extended screen which top edges are in the top monitor, I think the hot corners should be there. The idea is that a hot corner is easily reachable by just "throwing" the pointer over it, being between two screens doesn't help reaching the corner. Also, if I configured my top  monitor as the primary display (which I don't want to) the same would apply for the bottom corners.

**Comment by memeplex** [comment]
Another option could be what gnome does for the activities overview: if the pointer is against the sides of the screen it won't pass through to the top, so you can just throw the mouse towards a side and then up.

**Comment by JosephMcc** [comment]
Yeah, there is a pointer barrier on the ends of the panel to help with that but not blocking vertical movement.

**Comment by memeplex** [comment]
I noticed this also affects edge tiling, maybe I should open another issue, or do you prefer to generalize this one to something like "Multi-monitor edge resistance" or whatever.

**Comment by codewithmichael** [comment]
The above PR (#9550) fixes the issue on Cinnamon 4.4.8 (mint19).
